rubrik_cdm/exceptions.py

```diff
@@ -0,0 +1,18 @@
+class RubrikError(Exception):
```

Comment:
What's the purpose of this base class?

Reply:
The custom exceptions need to be derived from the `Exception` class. Standard practice (including the example in the Python documentation) had this being done indirectly though a base class. 

Doing it this way gives us more flexibility down the line in terms of being able to add some custom error message format logic in the base class as well. Something similar to `StripeError` here https://github.com/stripe/stripe-python/blob/master/stripe/error.py for example.